docs/services/vision/classification.md

```diff
@@ -21,140 +21,181 @@ The returned classifications consist of the image's class label and confidence s
 * `confidence` (float): specifies the confidence of the assigned label.
   Between `0.0` and `1.0`, inclusive.
 
-## Classifier Types
-
 The types of classifiers supported are:
 
 * **tflite_classifier**: a machine learning classifier that returns a class label and confidence score according to the specified `tensorflow-lite` model file available on the robot’s hard drive.
 
-### TFLite classifier
+## Configure a TFLite classifier
+
+### Create the ML Model Service for the classifier
+
+Navigate to the [robot page on the Viam app](https://app.viam.com/robots), then create an ML Model Service for the classifier model:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+Click on the robot you wish to add the classifier to.
+Select the **config** tab, and click on **Services**.
+
+Scroll to the **Create Service** section:
+
+1. Select `ml_model` as the **Type**.
```

Comment:
the type is just "mlmodel" rather than "ml_model"

It's a bit confusing, because the model in the vision service is called "ml_model" ...

Reply:
and that is staying? Because I feel like it should both be the same...

Reply:
fixed in the code so everything is "mlmodel"